kong/plugins/response-transformer/handler.lua

```diff
@@ -14,7 +14,8 @@ end
 
 function ResponseTransformerHandler:access(conf)
   ResponseTransformerHandler.super.access(self)
-  ngx.ctx.buffer = ""
+  ngx.ctx.chunks = {}
+  ngx.ctx.chunk_number = 1
```

Comment:
though it also applied on the original code, my comment would be to use a better namespaced name for these variables. The `ngx.ctx` table is shared for the lifetime of the request and hence has a risk of name colissions, especially with very generic names as `chunks`.

So maybe `ngx.ctx.rt_body_chunks` ?

Reply:
renamed

Reply:
By the way, perhaps, better kind of namespacing is to create a separate table for each plugin.
But this is breaking change and need to be tested hard if plugins access shared data.